app/_src/policies/meshloadbalancingstrategy.md

```diff
@@ -300,12 +302,13 @@ spec:
               - key: k8s.io/node
               - key: k8s.io/az
 ```
+{% endpolicy_yaml %}
 
-Apply the configuration with `kubectl apply -f [..]`.
+### Disable cross zone traffic and route to the local zone instances equally
 
-{% endtab %}
-{% tab local-zone-affinity-backend Universal %}
+In this example, when a user sends a request to the backend service, the request is routed equally to all instances in the local zone. If there are no instances in the local zone, the request will fail because there is no cross-zone traffic.
```

Comment:
nitpick: "cross zone" in the title of subsection, but "cross-zone" in the contents, should we stick with one way? Amazon uses "cross-zone" - https://docs.aws.amazon.com/elasticloadbalancing/latest/userguide/how-elastic-load-balancing-works.html